### ISSUE TITLE
Feature User Graduation System

### ISSUE THREAD

**Issue by MeesvanStraten** [DigitalExcellence/dex-backend#287]
**Is your feature request related to a problem? Please describe.**
Create a system to track when a user/ student graduates. This system should convert / create a new account with status alumni.
This alumni role will be a seperate issue. The implementation is done after the research is completed (issue #269)

This issue will implement the cmd application responsible for communicating with the api and delivering data to the message broker. The issue implementing a new alumni role is #288

**Additional context**
Use the user graduation research.

**Comment by niraymak** [comment]
This issue is started but is dependent on issue #256 

Blocked till mentioned issue is finished.

**Comment by niraymak** [comment]
Issue #288 is merged into this issue because these issues rely on eachother.
